src/textual/css/styles.py

```diff
@@ -551,6 +551,22 @@ def _align_size(self, child: tuple[int, int], parent: tuple[int, int]) -> Offset
             self._align_height(height, parent_height),
         )
 
+    @property
+    def partial_rich_style(self) -> Style:
+        """Get the style properties associated with this node only (not including parents in the DOM).
```

Comment:
Perhaps use a more property-style docstring here rather than a function style?

Reply:
Do you mean the "int: description"?

I've found that mkdocstrings doesn't render them correctly. It just omits the type :-(

Reply:
Yes and no. While that'd be ideal (and it does render really nicely in Emacs and VSCode hints, from what I've seen), I'm meaning more the wording. `"Get..."` sort of describing it as a function that does something, rather than as a property that has a value that has a particular meaning. So in this case likely just `"The style properties associated with this node only (not including parents in the DOM)`".

It's unfortunate that mkdocstrings doesn't handle it given it's, as far as I can tell, a documented aspect of the Google docstring convention.

PS: Just omits or just emits?

Reply:
The type in properties docstrings is just omitted. You see the description but no type.

With a function style, you do see the type.

Will assume that's a bug in mkdocstrings, and go for property style docs then.

Reply:
I may see about raising that with mkdocstrings then; it's unfortunate it does that.